infra/modules/azure_app_service_plan_autoscaler/variables.tf

```diff
@@ -105,23 +105,69 @@ variable "scale_metrics" {
       lower_threshold = optional(number, 20)
       increase_by     = optional(number, 1)
       decrease_by     = optional(number, 1)
-      }), {
-      upper_threshold = 80
-      lower_threshold = 20
-      increase_by     = 1
-      decrease_by     = 1
-    })
+    }), {})
     memory = optional(object({
       upper_threshold = optional(number, 70)
       lower_threshold = optional(number, 20)
       increase_by     = optional(number, 1)
       decrease_by     = optional(number, 1)
-      }), {
-      upper_threshold = 70
-      lower_threshold = 20
-      increase_by     = 1
-      decrease_by     = 1
-    })
+    }), {})
+    cooldown = optional(object({
+      requests_rule = optional(object({
+        increase = optional(string, "PT1M")
+        decrease = optional(string, "PT10M")
+      }), {})
+      cpu_rule = optional(object({
+        increase = optional(string, "PT1M")
+        decrease = optional(string, "PT20M")
+      }), {})
+      memory_rule = optional(object({
+        increase = optional(string, "PT1M")
```

Comment:
I suggest to ask for a number and hardcoding `PT` and `M` into the module, for developers would be easier to manage